extra_docs/ums/README.md

```diff
@@ -1,8 +1,14 @@
 # Jexia User Management System Module (UMS)
-UMS provides an ability to create user accounts, sign-in to the Jexia project with these accounts and perform 
-data changes with permissions of any authorized user. Users can be created either 
-with [Web Management Application](https://docs.jexia.com/getting-started/user-management/) or 
-with `signup()` method of this module.  
+UMS provides an ability to manage user accounts, sign-in to the Jexia project with these accounts and perform 
+data changes with permissions of any authorized user. There are a few ways to create a new user: 
+
+1. You can use [Web Management Application](https://docs.jexia.com/getting-started/user-management/) and create
+users one by one with nice UI. 
+2. You can use `signup()` method of User Management Module
+3. It is also possible to insert users as a regular data into UMS (the same way as dataset).
```

Comment:
It might not be the case... Most probably we will use SignUP with accepting some data.